<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть новый документ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать новый документ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>CTRL+O</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_4</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>F3</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_4</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>F6</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>CTRL+S</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_3</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>F2</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>CTRL+N</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_7">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>F12</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>84</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>CTRL+Q</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_4"/>
 </buttongroups>
</ui>
